docs/cli.md

```diff
@@ -29,17 +34,303 @@ Common Flags
+|`--env` (`-e`) *k1=v1,...*    | Set env vars *k1...* to values *v1...*             |
+|`--name` *name*               | Give *name* to all generated app artifacts         |
+|`--strategy` *s*              | Use build strategy *s*, one of: `docker`, `source` |
+|`--labels` (`-l`) *k1=v1,...* | Label all resources with *k1=v1,...*               |
+|`--insecure-registry`         | Bypass cert checks for referenced Docker images    |
+|`--output` (`-o`) *format*    | *format* is one of: `json`, `yaml`, `template`, `templatefile` |
+|`--output-version` *version*  | Output with *version* (default api-version)        |
+|`--no-headers`                | For default output, don't print headers            |
+|`--output-template` *s*       | Template string (`-o template`) or path (`-o templatefile`) |
+
+The template format is [golang templates](http://golang.org/pkg/text/template/#pkg-overview).
+The following example uses a MySQL image in a private registry to create an app and override application artifacts' names.
+
+```
+$ oc new-app \
+  --docker-image=myregistry.com/mycompany/mysql \
+  --name=private
+```
+
+See also [`oc start-build`](#oc-start-build).
+
+### oc status
+
+This shows a high level overview of the current project.
+
+See also [`oc describe`](#oc-describe) and [`oc get`](#oc-get).
+
+### oc project
+
+This displays the current project, or switches to another one.
+
+For example:
+
+```
+# Switch to the myapp project
+$ oc project myapp
+
+# Display the project currently in use
+$ oc project
+```
+
+## Build and Deploy Commands
+
+### oc start-build
+
+This manually starts a build, using either the specified buildConfig or a build name with the `--from-build` option.
+There is also the option of streaming
+the logs of the build if the `--follow` flag is specified.
+
+```
+$ oc start-build ruby-sample-build
+$ oc start-build --from-build=ruby-sample-build-275d3373-c252-11e4-bc79-080027c5bfa9
+$ oc start-build --from-build=ruby-sample-build-275d3373-c252-11e4-bc79-080027c5bfa9 --follow
+```
+
+See also [`oc new-build`](#oc-new-build) and [`oc new-app`](#oc-new-app).
+
+### oc build-logs
+
+This retrieves the logs from a Build container.
+It allows you to
+debug broken Build.
+If the build is still running, this streams the logs from the container to console.
+Use `oc get builds` to obtain a list of builds.
+
+```
+$ oc build-logs rubyapp-build
+```
+
+### oc deploy
+
+This views, starts, cancels or retries deployments.
+The general form is:
+
+```
+$ oc deploy <deployment-config> [options]
+```
+
+If invoked without options, display the latest deployment for the specified *deployment-config*.
+For example:
+
+```
+$ oc deploy database
+```
+
+Flag `--latest` starts a deployment.
+Flag `--retry` retries the latest failed deployment.
+Flag `--cancel` cancels the in-progress deployment.
+The following example shows how to cancel the deployment mentioned in the preceding example:
```

Comment:
@soltysh Good point.  Addressed as part of: https://github.com/tnguyen-rh/origin/commit/a7b6a2e7c03bf2777d15a1cd2230d2fd0c2ec968